### PR TITLE
Charts Update

### PR DESCRIPTION
**Pull request by thomas-tosoni**

### CODE REVIEW

File: app/src/core/campaigns/report/BasicReportCampaignController.js [hunk 1]
```diff
@@ -1,4 +1,3 @@
-/* global _, moment */
```
Comment (dduponchel) [inline app/src/core/campaigns/report/BasicReportCampaignController.js]: JSHint gives without that line a lot of `'_' is not defined.`.

File: app/src/core/campaigns/report/CampaignAnalyticsReportService.js [hunk 2]
```diff
@@ -1,31 +1,27 @@
-/* global _ */
```
Comment (dduponchel) [inline app/src/core/campaigns/report/CampaignAnalyticsReportService.js]: JSHint gives without that line a lot of `'_' is not defined.`.

File: app/src/core/campaigns/report/BasicReportCampaignController.js [hunk 3]
```diff
@@ -111,38 +106,72 @@ define(['./module'], function (module) {
...
 
       $scope.getDataForRow = getDataForRow;
 
+      /**
+       * X Axis Ticks
+       */
+
+      $scope.xAxisTicks = function () {
+        if ($scope.dateRangeIsToday()) {
+          var tickValues = [];
```
Comment (dduponchel) [inline app/src/core/campaigns/report/BasicReportCampaignController.js]: You should move this declaration above the `if`. The hoisting will create some hard-to-find bugs here.

**Comment by thomas-tosoni** [comment]
Fixed. Imported lodash instead of  declaring it globally.

**Pull request merged**